python/pyspark/sql/connect/plan.py

```diff
@@ -80,9 +80,19 @@ def _verify(self, session: "RemoteSparkSession") -> bool:
 
         return test_plan == plan
 
-    def collect(
```

Comment:
Let's problably fix the error here together.

Reply:
errors should have been fixed.